src/server/scripts/Northrend/Ulduar/Ulduar/boss_hodir.cpp

```diff
@@ -415,7 +416,7 @@ class boss_hodir : public CreatureScript
                         me->PlayDirectSound(SOUND_HODIR_FLASH_FREEZE, 0);
                         SmallIcicles(false);
                         events.RepeatEvent(55000 + urand(0,10000));
-                        events.ScheduleEvent(EVENT_SMALL_ICICLES_ENABLE, 12000);
+                        ((InstanceMap*)sMapMgr->FindMap(me->GetMapId(), me->GetInstanceId()))->GetMaxPlayers() > 15 ? events.ScheduleEvent(EVENT_SMALL_ICICLES_ENABLE, 12000) : events.ScheduleEvent(EVENT_SMALL_ICICLES_ENABLE, 24000);
```

Comment:
//events.ScheduleEvent(EVENT_SMALL_ICICLES_ENABLE, 12000);
events.ScheduleEvent(EVENT_SMALL_ICICLES_ENABLE, Is25ManRaid() ? 12000 : 24000);

Reply:
why do not？Check the number of players per play?

Reply:
The translation software may be inaccurate

Reply:
@Darkon47

Reply:
That would be a more efficient way of handling it, but for some reason it doesn't come up in my code completion as an option? just had the time to test it and it compiles fine, so I see no issue with using this more efficient version.